internalshared/configutil/telemetry.go

```diff
@@ -168,16 +174,32 @@ func parseTelemetry(result *SharedConfig, list *ast.ObjectList) error {
 		result.Telemetry.PrometheusRetentionTime = PrometheusDefaultRetentionTime
 	}
 
+	if result.Telemetry.UsageGaugePeriodRaw != nil {
+		var err error
+		if result.Telemetry.UsageGaugePeriod, err = parseutil.ParseDurationSecond(result.Telemetry.UsageGaugePeriodRaw); err != nil {
```

Comment:
I took another look at VLT-099; one thing we need is a way to disable this feature entirely.

My suggested solution was to accept "none" as a string, and set the gauge period to zero.  But if that's infeasible we can just document "0" as the special value.

Reply:
It seems easy enough to check for none right here.